Source/App/EbAppContext.c

```diff
@@ -381,6 +383,13 @@ EB_ERRORTYPE AllocateInputBuffers(
         callbackData->inputBufferPool->sliceType   = EB_INVALID_PICTURE;
     }
 
+    if (callbackData->ebEncParameters.segmentOvEnabled) {
+        size_t pictureWidthInLcu=(config->sourceWidth + EB_SEGMENT_BLOCK_SIZE - 1) / EB_SEGMENT_BLOCK_SIZE;
+        size_t pictureHeightInLcu=(config->sourceHeight + EB_SEGMENT_BLOCK_SIZE - 1) / EB_SEGMENT_BLOCK_SIZE;
+        size_t lcuTotalCount = pictureWidthInLcu * pictureHeightInLcu;
+        EB_APP_MALLOC(SegmentOverride_t*, callbackData->inputBufferPool->segmentOvPtr, sizeof(SegmentOverride_t) * lcuTotalCount, EB_N_PTR, EB_ErrorInsufficientResources);
```

Comment:
Need to free them as well, in the DeInitEncoder() procedure.

Reply:
As far as i understand the code and usage of EB_APP_MALLOC/EB_MALLOC, if I will use those macro,  allocator will free all allocated memory automatically in DeInitEncoder(). 

Am I right?

Reply:
I found it that EbDeinitEncoder() will free all the recorded memory resources.